src/client/components/SuggestedPage/style.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../style';
+
+.suggested-page {
+  background: $whitecolor;
+  box-shadow: 0 2px 4px 0 $shadowcolor;
+  font-size: .9em;
+  margin-right: 1em;
+  margin-top: 0;
+  padding: .2em 1.2em 1em 2em;
```

Comment:
When you write a line like this you should stop, step back and think through the css structure. Maybe you will find a better/cleaner solutions.